lib/fdk/runner.rb

```diff
@@ -98,9 +106,9 @@ def self.handle_function(function, req, resp)
 
     # TODO: gimme a bit me flexibility on response handling
```

Comment:
Would be nice to have the context split to InvokeContext and HTTPGateway context. At this moment the key difference is in headers, InvokeContext doesn't decap headers (decap from `Fn-Http-H-` prefix). But an HTTPGateway context does. Also, context attributes also changed, no more `FN_TYPE`, new headers:  `FN_ID`, etc.

Reply:
It's done

Reply:
see lib/fdk/context.rb HTTPContext

Reply:
I was looking for that on the master branch, but it's on the different one. Now i see that.